README.md

```diff
@@ -62,6 +63,7 @@ By definition, members work on their respective squad, although they are free to
 | Health Management             | David, João                      |    2/2       | [#health-mgmt-public](https://babylonhealth.slack.com/archives/CCNHJUXLH) |
 | Monitor                       | Anders, Michal                   |    2/2       | [#patient-metrics-team](https://babylonhealth.slack.com/archives/CE37S5W9Z) |
 | Triage UI                     | Michael                          |    1/2       | [#squad_triage_ui](https://babylonhealth.slack.com/archives/CE6H6SLRX) |
+| Women's health                | Anil                             |    1/1       | [#womenshealthteam](https://babylonhealth.slack.com/archives/CFX73S4P5) |
```

Comment:
```suggestion
| Women's Health                | Anil                             |    1/1       | [#womenshealthteam](https://babylonhealth.slack.com/archives/CFX73S4P5) |
```

Reply:
[ ❔] Question (especially since I, myself, didn't understand that column the first time I read this table): is this `1/1` confirmed? I have no idea if we expect to fill more positions for this squad (and if @anilputtabuddhi put `1/1` here without knowing about it maybe), or if the squad positions are really all addressed?

Reply:
@AliSoftware I assumed that it was the (available/required) ios numbers in the squad. I was told there will be one iOS and one Back end dev in the near future but not too sure about the long term plans. @RuiAAPeres @dmcrodrigues Could you please help with this?

Reply:
@anilputtabuddhi To my understanding it is indeed the available/required iOS numbers in the squad. I think if you were told that there will only be one iOS in the near future, then it's ok to keep `1/1` then 👍
We can always amend with another PR if long term plans change, don't worry about it 😉

Reply:
At the moment this is accurate.